### PR TITLE
Adding basic skin + screens

### PR DESCRIPTION
**Pull request by John-Kirk**

### CODE REVIEW

File: core/src/com/nerds/gamejam/screen/StartScreen.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+        addPaddingRow(table, 3);
+        table.add().expand();
+        table.add(innerTable);
+        table.add().expand();
+        table.row();
+        addPaddingRow(table, 3);
+
+        table.setFillParent(true);
+//        table.debug();
```
Comment (Unequity) [inline core/src/com/nerds/gamejam/screen/StartScreen.java]: Probably want to remove

File: core/src/com/nerds/gamejam/screen/StartScreen.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+    private Button createResumeButton() {
+        Button resumeButton = new TextButton("Continue Game", skin);
+        resumeButton.setTouchable(Touchable.disabled);
+        return resumeButton;
+    }
+
+    private Button createExitButton() {
+        Button quitButton = new TextButton("Quit Game", skin);
+        quitButton.addListener(new ClickListener() {
```
Comment (Unequity) [inline core/src/com/nerds/gamejam/screen/StartScreen.java:90]: Can this be lambda'd?

**Comment by Unequity** [comment]
I'm guessing preview.png doesn't need to be there as it's not part of the actual asset pack (I can't comment directly on the file)

File: core/src/com/nerds/gamejam/screen/StartScreen.java (hunk 2 above)
Comment (John-Kirk) [reply core/src/com/nerds/gamejam/screen/StartScreen.java:90]: Nah, ClickListener has like a billion methods on it, so no lambdas :(

Comment (Unequity) [reply core/src/com/nerds/gamejam/screen/StartScreen.java:90]: SAD times :(

File: core/src/com/nerds/gamejam/screen/GameScreen.java [hunk 3]
```diff
@@ -23,10 +23,10 @@ public GameScreen(GameJam game, Stage stage, Skin skin) {
 
     @Override
     public void show() {
-        Table table = new Table();
-        table.add(new Label("Hello there!", skin)).expand();
-        table.setFillParent(true);
-        stage.addActor(table);
+        ScrollingBackground scrollingBackground = new ScrollingBackground(new Texture("stars.png"));
```
Comment (Unequity) [inline core/src/com/nerds/gamejam/screen/GameScreen.java]: Seems weird to set the scrolling background again here - means it jumps to the initial position again - I think having the "hello there" placeholder was good enough given we'll be presumably replacing this with a character selection screen?

Comment (lozyness) [reply core/src/com/nerds/gamejam/screen/GameScreen.java]: boooo Daniel! We should have had a rule against all boring bastards

**Pull request merged**